data/WOTC_5e_SRD_v5.1/armor.json

```diff
@@ -0,0 +1,187 @@
+[
+    { "id": "no_armor",
+      "name": "No Armor",
+      "type": "No Armor",
+      "rarity": "Standard",
+      "ac": 10,
+      "cost": "5 gp",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": false
+    }, {
+      "id": "padded_armour",
+      "name": "Padded Armor",
+      "type": "Light Armor",
+      "rarity": "Standard",
+      "ac": 11,
+      "cost": "5 gp",
+      "weight": "8 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": true
+    }, {
+      "id": "leather_armour",
+      "name": "Leather Armor",
+      "type": "Light Armor",
+      "rarity": "Standard",
+      "ac": 11,
+      "cost": "10 gp",
+      "weight": "10 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": false
+    }, {
+      "id": "studded_leather_armour",
+      "name": "Studded Leather Armor",
+      "type": "Light Armor",
+      "rarity": "Standard",
+      "ac": 12,
+      "cost": "45 gp",
+      "weight": "13 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": false
+    }, {
+      "id": "hide",
+      "name": "Hide",
+      "type": "Medium Armor",
+      "ac": 12,
+      "cost": "10 gp",
+      "weight": "12 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": false
+    }, {
+      "id": "chain_shirt",
+      "name": "Chain Shirt",
+      "type": "Medium Armor",
+      "rarity": "Standard",
+      "ac": 13,
+      "cost": "50 gp",
+      "weight": "20 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": false
+    }, {
+      "id": "scale",
+      "name": "Scale mail",
+      "type": "Medium Armor",
+      "rarity": "Standard",
+      "ac": 14,
+      "cost": "50 gp",
+      "weight": "45 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": true
+    }, {
+      "id": "breastplate",
+      "name": "Breastplate",
+      "type": "Medium Armor",
+      "rarity": "Standard",
+      "ac": 14,
+      "cost": "400 gp",
+      "weight": "20 lb.",
+      "modifier": [ "dexterity" ],
```

Comment:
On these guys I think this is supposed to be dex (max 2). Is there any way to indicate that here?

Reply:
what do you think about about add `"modifier_max": 2`

Reply:
This is the code to calculate ac using that data source

https://github.com/gbalbuena/dndcli/blob/master/src/core/index.js#L3-L43